BrainPortal/public/javascripts/userfiles.js

```diff
@@ -441,49 +483,6 @@ $(function() {
 
     /* Show/Hide dynamic actions/menu elements according to current selection */
     (function () {
-      // Generate the launch_task div if prepare_tool_id present as a parameter
-      function launch_task() {
```

Comment:
So what happened here? This function was moved? Was it necessary?

And what happens to the launch_task() call at line 499 ? It's no longer in scope I think?

Reply:
I moved the function  in order to be able to call it at 2 different places:
- at line 56 in `cbrain_attach_userfile_checkboxes` (before some specific (buggy) code was called to update the number of files)
- at line 499 as before ( I test it and the launch_task function is reachable). 

This way I have a single function that is able to update the number of task selected, instead of having 2 differents pieces of code to do that.